### DISCUSSION TITLE
Seperate ODoH Servers from ODoH relays

### DISCUSSION THREAD

**Discussion by jumpsmm7** [DNSCrypt/dnscrypt-proxy#1736] [Ideas]
With the implementation for future support of ODoH servers and relays, I think it is crucial to not mesh servers and relays in the same .md file. Can the servers be separated from the relays?

**Comment by jedisct1** [comment]
Yes, this is a good idea, if only to be consistent with other relay types.

But right now, the number of entries is quite low. So, maybe we can keep a single file at least during the beta period.

**Reply by jumpsmm7** [reply]
Correct @jedisct1 , you and your team (anyone who has helped) have done a wonderful job streamlining this to a beta level. My idea is simply ironing out the kinks, like you said, to be consistent with other relay/server formating and prevent user confusion. Keep up the hard work.

**Reply by jumpsmm7** [reply]
> 
> 
> Yes, this is a good idea, if only to be consistent with other relay types.
> 
> But right now, the number of entries is quite low. So, maybe we can keep a single file at least during the beta period.

It may be better to leave it as one file, I actually made it work for my needs just as one file. I think it works great for me and the users of asuswrt-merlin routers. @jedisct1  Thank you for everything you do!.
